jte/src/main/java/gg/jte/compiler/IoUtils.java

```diff
@@ -18,17 +21,14 @@ public static String toString(InputStream inputStream) throws IOException {
     }
 
     public static void deleteDirectoryContent(Path directory) {
-        if (!Files.exists(directory)) {
-            return;
-        }
-
-        try {
+        Objects.requireNonNull(directory);
+        try (Stream<Path> pathStream = Files.walk(directory)) {
             //noinspection ResultOfMethodCallIgnored
-            Files.walk(directory)
-                    .filter(d -> d != directory)
+            pathStream.filter(d -> d != directory)
                     .sorted(Comparator.reverseOrder())
                     .map(Path::toFile)
                     .forEach(File::delete);
+        } catch (NoSuchFileException ignored) {
```

Comment:
`NoSuchFileException` looks a bit suspicious here. Maybe have a `deleteQuietly` method that is called for each file?

Reply:
I'll just revert this